CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ All notable changes to the "md-tags" extension will be documented in this file.
 
 Check [Keep a Changelog](http://keepachangelog.com/) for recommendations on how to structure this file.
 
+## v0.2.2
```

Comment:
I already published `v0.2.2` and I think you should expect a conflict in this file. You should use `Unreleased`.

```suggestion
## Unreleased
```

As this is a new feature, I may be published in `v0.3.0`.

Reply:
damn,  missed that